File: complexity/primeOrNot.cpp
#include<iostream>
using namespace std;

bool isPrime(int x){
    if(x<=1) return false;
    for(int i=2;i*i<x-1;i++){
        if(x%i==0){
            return false;
        }
    }
    return true;
}
int main()
{
    cout<<isPrime(3);
    return 0;
}

File: arrays/primeOrNot.js
function isPrime(n) {
  let result = true;
  
  if(n==1) return false
  for (let i = 2; i * i < n; i++) {
    if (n % i == 0) {
      result = false;
    }
  }
  return result;
}
console.log(isPrime(1));
